<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appearance</class>
 <widget class="QWidget" name="appearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="appearanceGroupBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="useThumbnails">
        <property name="text">
         <string>Use thumbnails instead of icons</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="KComboBox" name="titleLabelMode"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="customBackgroundPathLabel">
          <property name="text">
           <string>Custom background image:</string>
          </property>
          <property name="buddy">
           <cstring>customBackgroundImage</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KComboBox" name="activeIconIndication"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="titleLabelModeLabel">
          <property name="text">
           <string>Show title label:</string>
          </property>
          <property name="buddy">
           <cstring>titleLabelMode</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="activeIconIndicationLabel">
          <property name="text">
           <string>Indication for active icon:</string>
          </property>
          <property name="buddy">
           <cstring>activeIconIndication</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KUrlRequester" name="customBackgroundImage"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="animationsGoupBox">
     <property name="title">
      <string>Animations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLabel" name="startupAnimationLabel">
        <property name="text">
         <string>Starting tasks:</string>
        </property>
        <property name="buddy">
         <cstring>startupAnimation</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="demandsAttentionAnimationLabel">
        <property name="text">
         <string>Tasks needing attention:</string>
        </property>
        <property name="buddy">
         <cstring>demandsAttentionAnimation</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="moveAnimation"/>
      </item>
      <item row="5" column="1">
       <widget class="KComboBox" name="demandsAttentionAnimation"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="parabolicMoveAnimation">
        <property name="text">
         <string>Parabolic</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="moveAnimationLabel">
        <property name="text">
         <string>Moving between icons:</string>
        </property>
        <property name="buddy">
         <cstring>moveAnimation</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KComboBox" name="startupAnimation"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
